index.js

```diff
@@ -2,12 +2,11 @@
 
 const execa = require('execa')
 
-module.exports = async (input) => {
+module.exports = async (input, options) => {
```

Comment:
This line should be changed to have `options = { }` so if the user doesn't pass any options, the code doesn't throw an exception.

See the unit test [failure](https://travis-ci.org/transitive-bullshit/ffmpeg-probe/jobs/379413388).